Binary/Locator/FileSystemInsecureLocator.php

```diff
@@ -21,9 +21,9 @@ class FileSystemInsecureLocator extends FileSystemLocator
      */
     protected function generateAbsolutePath(string $root, string $path): ?string
     {
-        if (false === mb_strpos($path, '..'.DIRECTORY_SEPARATOR) &&
-            false === mb_strpos($path, DIRECTORY_SEPARATOR.'..') &&
-            false !== file_exists($absolute = $root.DIRECTORY_SEPARATOR.$path)
+        if (false === mb_strpos($path, '..'.\DIRECTORY_SEPARATOR) &&
```

Comment:
why the backslashes here?

Reply:
it is global constant and php-cs doesn't like it without backslash

Reply:
was fixed in [1107](https://github.com/liip/LiipImagineBundle/pull/1107).
will remove backslashes.